src/transaction/declare_v2.rs

```diff
@@ -386,7 +366,7 @@ mod tests {
             chain_id,
             sender_address,
             0,
-            0.into(),
```

Comment:
Shouldn't this depend on the Cairo contract's version (1 or 2)?

Reply:
Yes, in this case the contract is Sierra so it should be v1

Reply:
Yes you are right, I will fix this